src/audio/google_rtc_audio_processing_mock.c

```diff
@@ -45,6 +54,35 @@ GoogleRtcAudioProcessingState *GoogleRtcAudioProcessingCreate()
 	return s;
 }
 
+GoogleRtcAudioProcessingState *GoogleRtcAudioProcessingCreateWithConfig(int capture_sample_rate_hz,
+									int num_capture_input_channels,
+									int num_capture_output_channels,
+									int render_sample_rate_hz,
+									int num_render_channels,
+									const uint8_t *const config,
+									int config_size)
+{
+	struct GoogleRtcAudioProcessingState *s =
+		rballoc(0, SOF_MEM_CAPS_RAM, sizeof(GoogleRtcAudioProcessingState));
+	if (!s)
+		return NULL;
+	s->num_capture_channels = num_capture_input_channels;
+	s->num_aec_reference_channels = num_render_channels;
+	s->num_output_channels = num_capture_output_channels;
+	s->num_frames = capture_sample_rate_hz * 10 / 1000;
```

Comment:
what is 10/1000? is it a time conversion?

Reply:
10 is the frame size in ms, and the computation produces in the frame size in samples.
This now changed this to make it more clear.

Reply:
Might be worth defineing that as it is not inherently clear

Reply:
Done.